recipes/tardis/meta.yaml

```
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
```

I think this package supports Python 2 but if not you can constrain it using:
```suggestion
  skip: true  # [not win or py2k]
```

---

Yes, it does support Python 2.